docs/Operators.md

```diff
@@ -2349,6 +2357,45 @@ for test_case, values_ in test_cases.items():
 </details>
 
 
+### <a name="ConcatFromSequence"></a><a name="concatfromsequence">**ConcatFromSequence**</a>
+
+  Concatenate a sequence of tensors into a single tensor.
```

Comment:
IIUC, this corresponds to `np.concatenate`. I wonder if it makes sense to support `np.stack` as well. `np.stack` could be implemented by `Loop`-ing for the input sequence, `Unsqueeze` in each tensor in the sequence, and `ConcatFromSequence`, but I think this is a bit tedious. I think one solution is adding an attribute like `new_axis=false`. If this is true, this operator behaves like `np.stack`

Reply:
Thanks! That makes sense, adding `new_axis` as attribute default to `0` to support both `concat` and `stack`.